ui/src/datasets.tsx

```diff
@@ -126,11 +126,9 @@ export function Datasets() {
     conceptSets: { id: string; name: string }[]
   ) => {
     // TODO(tjennison): Only show demographics until we have better prepackaged
+    // Updated for AoU to show all prepackaged configured in the top_level.json
```

Comment:
We don't want to show the Analgesics prepackaged concept set everywhere so if you want to remove the splice here you'll need to remove Analgesics from any of the other underlays it appears in.

Also, just remove this entire comment now that it's not applicable anymore.

Reply:
done